src/components/LogRow/RowRenderer/index.tsx

```diff
@@ -38,6 +40,24 @@ const ParsleyRow: RowRendererFunction = ({ logType, processedLogLines }) => {
       ? new RegExp(`${highlights.map((h) => `(${h})`).join("|")}`, "gi")
       : undefined;
 
+  let Row: React.FC<ResmokeRowProps> | React.FC<AnsiRowProps>;
```

Comment:
This is more flexible and avoids exporting the the Row props.
```suggestion
  let Row: typeof ResmokeRow | typeof AnsiRow;
```